### ISSUE TITLE
Please add an option to use a timestamp as input time format

### ISSUE THREAD

**Issue by FlorentMasson** [mcastorina/graph-cli#44]
Please also add the link to possible date formats in the docs. I guess it's this? https://docs.python.org/3/library/datetime.html#strftime-and-strptime-behavior

**Comment by mcastorina** [comment]
Yes, great idea

**Comment by mcastorina** [comment]
> Please add an option to use a timestamp as input time format

Can you please provide an example? Do you mean a unix timestamp (seconds since epoch)?

**Comment by FlorentMasson** [comment]
Yes, I'm referring to unix timestamp for files like this:

```
timestamp,value
1616976024321,15
1616976084322,16
1616976144322,13
1616976204323,12
1616976264323,15
1616976324324,23
1616976384324,25
1616976444324,12
1616976504325,18
1616976564325,22
```

In this file the timestamp is the number of milliseconds as returned by java `System.currentTimeMillis()`

**Comment by mcastorina** [comment]
Great, thanks for the example. I will probably make a special value for `--time-format-input` that will interpret it as seconds / ms since epoch. I think `epoch` and `epoch-ms` would be good, what do you think?

Alternatively, `epoch` could try to guess whether it's seconds or milliseconds automatically.

**Comment by FlorentMasson** [comment]
That would be great in either case!
Yes that could be done automatically as a current ms timestamp interpreted as seconds gives year 53209. With a high-enough threshold, we have plenty of time before it breaks 😆

**Comment by mcastorina** [comment]
Implemented in 10837cb4efe49907468c57d9d878e37437e3faf6 and available in version v0.1.10

Using `epoch` will first try to convert the column as seconds, then try milliseconds.

**Comment by FlorentMasson** [comment]
That's just... awesome! Working great, thanks!
